src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -20783,10 +20783,22 @@ public class ClassC { }
 }";
             var referenceD = CreateCompilation(codeD, assemblyName: "D").EmitToImageReference();
 
-            CompileAndVerify(
+            var verifier = CompileAndVerify(
                 source: codeA,
                 references: new MetadataReference[] { referenceB, referenceC2, referenceD },
-                expectedOutput: "obj is null");
+                expectedOutput: "obj is null",
+                verify: Verification.FailsILVerify with { ILVerifyMessage = "[Main]: Unable to resolve token. { Offset = 0x1, Token = 167772167 }" });
```

Comment:
Filed an issue on the runtime repo for this regression. See issue [https://github.com/dotnet/runtime/issues/96702](https://github.com/dotnet/runtime/issues/96702) for details

Reply:
Looks like this issue on runtime was resolved. Should we just add a comment here that it's a known issue and link to the discussion?

Reply:
I added a comment above with ECMA rule explaining verification failure